### ISSUE TITLE
Setup `dotnet test` output reporter to GitHub Actions

### ISSUE THREAD

**Issue by Aaronontheweb** [akkadotnet/Akka.Hosting#200]
https://github.com/phoenix-actions/test-reporting - if we can format the test output from `dotnet test` and copy it into a folder (akin to what we do with AzDo for most other Akka.NET projects) we can use this to provide detailed test reporting when unit tests in this repository fail.

**Comment by SeanFarrow** [comment]
Are you still looking to implement this? If yes, I'd love to have a look at doing this in the next few weeks.
Thanks,
Sean.

**Comment by Aaronontheweb** [comment]
Yes we are! I've been able to implement this in some other repos: https://seankilleen.com/2024/03/beautiful-net-test-reports-using-github-actions/ - that or something similar to it would be great
